openassessment/xblock/submission_mixin.py

```diff
@@ -376,6 +376,7 @@ def submission_path_and_context(self):
         """
         workflow = self.get_workflow_info()
         problem_closed, reason, start_date, due_date = self.is_closed('submission')
```

Comment:
(this applies to each of the files we've called out for time zone fetching)

I think a good way to treat this is "If and when dates are defined, I should fetch the time zone"